docker-compose.yml

```diff
@@ -78,7 +78,7 @@ services:
       # For headless delve
       - ${PMM_PORT_DELVE:-2345}:2345
       # PG
-      - ${PMM_PORT_PG:-15432}:5432
+      - ${PMM_PORT_PG:-5432}:5432
```

Comment:
It was probably intentional, otherwise it'd get in conflict with https://github.com/percona/pmm/blame/main/docker-compose.yml#L192

Reply:
I was testing locally and forgot to revert it. Actually it shouldn't conflict with that one because they are in 2 different profiles and shouldn't run together.